.github/workflows/run-clang-format.sh

```diff
@@ -16,7 +16,7 @@
 FORMAT_OUT=${TMPDIR:-/tmp}/clang-format-diff.out
 
 # Run on all the files that are affected
-clang-format -i --style=file $(git diff --name-only --diff-filter=AM -r origin/master | grep '\(\.cc\|\.h\)$') 2> /dev/null
+clang-format -i --style=Google $(git diff --name-only --diff-filter=AM -r origin/master | grep '\(\.cc\|\.h\)$') 2> /dev/null
```

Comment:
`git diff --name-only --diff-filter=AM -r origin/master` <-- Using `origin/master` there is most certainly wrong....

Reply:
the `GITHUB_BASE_REF` environment variable is also `master` (which is our main branch from which we branch off in a pull request). So it would be exactly the same (but we have to prepend origin/).

So we could do something like `BASE_BRANCH=origin/${GITHUB_BASE_REF:-master}`, but the invocation of `git diff` is exactly the same.

Reply:
I am now explicitly determining the point from where it forked off:
```
FORK_POINT=$(git merge-base --fork-point ${BASE_BRANCH})
```